attributes/default.rb

```diff
@@ -107,3 +108,7 @@
 default['hopsmonitor']['prometheus']['kube-crt']         = ""
 default['hopsmonitor']['prometheus']['kube-key']         = ""
 default['hopsmonitor']['prometheus']['kube-ca']          = ""
+
+# Configuration attributes for alerting rules
+default['prometheus']['rules']['all-hosts-available-mem-threshold']  = "10"
+default['prometheus']['rules']['ndbd-hosts-available-mem-threshold'] = "3"
```

Comment:
btw, for app we set this to `1.5%` is `3%` a realistic default?

Reply:
Well that's something very specific to the machine's specs. A VM with 250G of RAM with the default RonDB memory configuration leaves 4G for OS. I changed this to `1.5%`